src/sqlancer/stonedb/gen/StoneDBTableAlterGenerator.java

```diff
@@ -27,19 +32,48 @@ public static SQLQueryAdapter generate(StoneDBGlobalState globalState) {
 
     private SQLQueryAdapter getQuery() {
         sb.append("ALTER TABLE ");
-        StoneDBTable table = globalState.getSchema().getRandomTable(t -> !t.isView());
-        StoneDBExpressionGenerator generator = new StoneDBExpressionGenerator(globalState)
-                .setColumns(table.getColumns());
         sb.append(table.getName());
         sb.append(" ");
-        Action action = Randomly.fromOptions(Action.values());
+        appendAlterOptions();
+        addExpectedErrors();
+        return new SQLQueryAdapter(sb.toString(), errors, true);
+    }
+
+    private void addExpectedErrors() {
```

Comment:
For future expected errors: I think it would be better to inline the expected errors. They should be as localized as possible, so we don't miss any bugs that we could otherwise easily catch.

Reply:
got it, will add errors immediately after the code that will cause such an error.